runtime/ops/manipulation.cc

```diff
@@ -82,14 +82,26 @@ chainerx::Array TransposeOp::RunImpl(XCVMState* st, const chainerx::Array& data)
 
 chainerx::Array PadOp::RunImpl(XCVMState* st, const chainerx::Array& data) {
     CHECK_EQ(data.ndim() * 2, pads.size());
-    chainerx::Shape shape = data.shape();
-    std::vector<chainerx::ArrayIndex> indices;
+    const auto shape = data.shape();
```

Comment:
Could you use `const chainerx::Shape`, please? Overall, could you try avoiding `auto`? Copy and paste from a previous comment:

Personally, I'm not a big fan of auto. IMHO, it sometimes hurts readability significantly while it only saves you a few key strokes. I think I allow myself to use auto only for complex types. Typical examples are STL iterators, lambdas, and smart pointers.

I know this is my personal preference, but I really don't like overused `auto`..

Reply:
I removed auto as possible.